### PR TITLE
minimum throttle-squared 3d quad trajectory 

### PR DESCRIPTION
**Pull request by cjmcgreal**
Added a new example.

### CODE REVIEW

File: demo/quadRotor3d/utilities/loadPlant_QuadRotor3d.m [hunk 1]
```diff
@@ -0,0 +1,30 @@
+function [p] = loadPlant_QuadRotor3d()
```
Comment (MatthewPeterKelly) [inline demo/quadRotor3d/utilities/loadPlant_QuadRotor3d.m:1]: Thanks for breaking this out into its own function, and for making nice documentation.

File: demo/quadRotor3d/README.md [hunk 2]
```diff
@@ -11,6 +12,7 @@ Dynamics:
 
 Objective:
     1. Minimum time solution for boundary value problem with control and kinematic limits.
+         See entry-point script: MAIN_minTimeTraj.m
+    2. Minimum throttle-squared solution for boundary value problem with control and kinematic limits.
+         See entry-point script: MAIN_minThrottleTraj.m
     TODO: add other objective functions
```
Comment (MatthewPeterKelly) [inline demo/quadRotor3d/README.md:18]: Can this line can be removed now?

File: demo/quadRotor3d/MAIN_minTimeTraj.m [hunk 3]
```diff
@@ -0,0 +1,83 @@
+% MAIN - Minimum Time Boundary Value Problem
```
Comment (MatthewPeterKelly) [inline demo/quadRotor3d/MAIN_minTimeTraj.m:1]: It looks like this script replaces the original `MAIN.m` script. Would you be able to remove that file now if it is unused?

**Review by MatthewPeterKelly — COMMENTED** [review]
These updates look great - thanks for adding them!  

Would you be able to upload some plots to that show the solution to the new optimization to this pull request?  (on GitHub, not in the code)

File: demo/quadRotor3d/utilities/loadPlant_QuadRotor3d.m (hunk 1 above)
Comment (cjmcgreal) [reply demo/quadRotor3d/utilities/loadPlant_QuadRotor3d.m:1]: > These updates look great - thanks for adding them!
> 
> Would you be able to upload some plots to that show the solution to the new optimization to this pull request? (on GitHub, not in the code)

I'm happy to! Can you elaborate on this? Upload screenshots / pngs? just right in the demo/quadRotor3d folder?

Comment (MatthewPeterKelly) [reply demo/quadRotor3d/utilities/loadPlant_QuadRotor3d.m:1]: GitHub has great interface for adding images into comments (both comments in the code and in the top-level PR).  Here is a simple example:  I took a screen shot and then dragged the image file into the comment box. GitHub will automatically upload it and format it. You can click "Preview" to see what it looks.
demo-plot

For this PR, you could make some plots and save them as .png files (either through Matlab or by using a screen shot). Then add a comment on the PR and drag the image files into that comment. This makes them easily visible to users online, but doesn't increase the size of the code that users need to clone to use the repository.

Comment (cjmcgreal) [reply demo/quadRotor3d/utilities/loadPlant_QuadRotor3d.m:1]: ok! Thanks for articulating!

**Comment by cjmcgreal** [comment]
Plots of minimum throttle 3d quadcopter example solution. Plots created with _plotQuadRotor3d.m_
states
rates
controls

**Comment by rhjiang** [comment]
MAIN_minThrottleTraj.m ran well for me!

**Comment by MatthewPeterKelly** [comment]
> Plots of minimum throttle 3d quadcopter

Thanks for adding the plots!  That's a nice reference point to have.

**Comment by MatthewPeterKelly** [comment]
> MAIN_minThrottleTraj.m ran well for me!

Thanks for testing the new code!

**Pull request merged**